pkg/analyzer/filter/eval.go

```
@@ -55,8 +60,37 @@ func NewEvaluator(name string, ignoreError bool) (Evaluator, error) {
 		cel.Variable(filterInputVarScorecard, cel.DynType),
 		cel.Variable(filterInputVarLicenses, cel.DynType),
 		cel.Variable(filterInputVarRoot, cel.DynType),
+		cel.Function("contains_license",
+			cel.MemberOverload("list_string_contains_license_string", []*cel.Type{cel.ListType(cel.StringType), cel.StringType}, cel.BoolType,
+				cel.BinaryBinding(func(lhs, rhs ref.Val) ref.Val {
+					l := lhs.(traits.Lister)
+					filterlicenseexp := fmt.Sprintf("%s", rhs)
+					iter := l.Iterator()
+					contains := false
+					i := 0
+					for {
+						if contains {
+							break
+						}
+						if iter.HasNext().Value() == true {
```

Fail fast is a strong engineering practice to follow. This means:

```
if !if iter.HasNext().Value() {
   break
}
```